### PR TITLE
*: add metric for FlushRowChangedEvents in sinkDriver

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What problem does this PR solve? 
Add metric

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: cdc/processor.go [hunk 1]
```diff
@@ -698,11 +699,13 @@ func (p *processor) sinkDriver(ctx context.Context) error {
 			if minTs == 0 {
 				continue
 			}
+			start := time.Now()
 			err := p.sink.FlushRowChangedEvents(ctx, minTs)
 			if err != nil {
 				return errors.Trace(err)
 			}
 			atomic.StoreUint64(&p.checkpointTs, minTs)
+			metricFlushDuration.Observe(time.Since(start).Seconds())
```
Comment (leoppro) [inline cdc/processor.go:708]: It's too granular in seconds.

Comment (overvenus) [reply cdc/processor.go:708]: It's a float64.

**Comment by amyangfei** [comment]
Please resolve conflicts

**Pull request merged**